simphony_ui/ui.py

```diff
@@ -203,20 +208,58 @@ def show_liggghts_result(self):
         self.liggghts_flow_source = dataset2cudssource(liggghts_flow_dataset)
         self.liggghts_wall_source = dataset2cudssource(liggghts_wall_dataset)
 
-        flow_modules = default_module(self.liggghts_flow_source)
-        wall_modules = default_module(self.liggghts_wall_source)
+        self._add_liggghts_source_to_scene(
+            liggghts_flow_dataset,
+            self.liggghts_flow_source
+        )
+        self._add_liggghts_source_to_scene(
+            liggghts_wall_dataset,
+            self.liggghts_wall_source
+        )
+
+    def _add_liggghts_source_to_scene(self, dataset, source):
+        mayavi_engine = self.mlab_model.engine
+
+        # Create Sphere glyph
+        sphere_glyph_module = Glyph()
 
         # Add Liggghts sources
-        mayavi_engine.add_source(self.liggghts_flow_source)
+        mayavi_engine.add_source(source)
+
+        source.point_vectors_name = 'VELOCITY'
+
+        # Add sphere glyph module
+        mayavi_engine.add_module(sphere_glyph_module)
+
+        sphere_glyph_module.glyph.glyph_source.glyph_source = \
+            SphereSource()
+        sphere_glyph_module.glyph.scale_mode = 'scale_by_scalar'
+        sphere_glyph_module.glyph.glyph.range = [0.0, 1.0]
+        sphere_glyph_module.glyph.glyph_source.glyph_source.radius = \
+            1.0
+
+        # Create Arrow glyph
+        # Get maximum particle velocity
+        velocities = numpy.array(
+            [numpy.linalg.norm(particle.data[CUBA.VELOCITY])
+             for particle
+             in dataset.iter_particles()]
+        )
+        max_velocity = numpy.max(velocities)
 
-        # Add default Liggghts modules
-        for module in flow_modules:
-            mayavi_engine.add_module(module)
+        # If the max velocity is not 0, we create the arrow glyph
+        if max_velocity != 0:
+            # Set scale factor of arrows
+            arrow_scale_factor = 50000.0
```

Comment:
which means that we are working in the range of tens of micrometers/sec. I would be tempted to say 1 gu = 1 micrometer/sec. That will make arrows probably good enough for all purposes.